profile/README.md

```diff
@@ -1,55 +1,14 @@
-![Alephium Logo](https://raw.githubusercontent.com/alephium/alephium-brand-guide/master/logos/light/Logo-Horizontal-Light.png#gh-dark-mode-only) ![Alephium Logo](https://raw.githubusercontent.com/alephium/alephium-brand-guide/master/logos/dark/Logo-Horizontal-Dark.png#gh-light-mode-only)
+## Hello World!
 
-### A new paradigm
+Alephium is the first operational sharded L1 blockchain scaling and enhancing PoW & UTXO concepts. To learn more about Alephium, visit [wiki.alephium.org](https://wiki.alephium.org/).
 
-Alephium is the first operational sharded L1 blockchain scaling and enhancing PoW & UTXO concepts. Decentralization, self-sovereignty, and security meet high-performance, accessibility, and energy efficiency in a dev-friendly network optimized for DeFi & smart contract applications.
+### Contributing to the Ecosystem
 
-### Accessibility & Usability
+Alephium has an early but growing ecosystem that welcomes all kinds of contributions.
 
-From its technical design to its interfaces, Alephium has been created to address the challenges of accessibility, scalability, and security encountered by decentralized applications today.
+- Start with the [quick guide](https://wiki.alephium.org/dapps/Getting-Started) and if you’re building a dApp.
+- Let everyone know by sending a PR to add your project to the [Awesome Alephium](https://github.com/alephium/awesome-alephium) repo!
+- The Alephium [branding guide](https://github.com/alephium/alephium-brand-guide) might also come in handy.
+- [Join our Discord](https://discord.gg/JErgRBfRSB) and ask the devs and a thriving community of Alephians for help.
 
-### Overview
-
-Alephium is fully open-source and development happens publicly. Contributions from anyone looking to improve and expand the codebase are welcomed and encouraged. 
-
-Alephium’s full node implementation is written in Scala and has a JS/TS SDK.  \
-For smart contract and dApp development, Alephium has its own language called [Ralph](https://wiki.alephium.org/ralph/built-in-functions).
-
-## Start here
-
-**Read the [documentation](https://wiki.alephium.org).  \
-Find the latest[ Node release](https://github.com/alephium/alephium/releases/latest/).  \
-Get the latest [Desktop Wallet.](https://github.com/alephium/desktop-wallet/releases/latest/) \
-Discover the [Whitepapers](https://github.com/alephium/white-paper). \
-Browse the [FAQ](https://wiki.alephium.org/frequently-asked-questions).**
-
-Discover and contribute to our main projects:
```

Comment:
I would leave the list of main repos, it's really not user friendly to navigate with the repo list + we want people to contribute to those as well, not just build Dapps